backend/src/Controllers/voterRolls.controller.ts

```diff
@@ -30,11 +30,6 @@ const getRollsByElectionID = async (req: any, res: any, next: any) => {
     }
 }
 
-const returnRolls = async (req: any, res: any, next: any) => {
-    Logger.info(req, `${className}.returnRolls ${req.params.id}`);
-    res.json({ election: req.election, electionRoll: req.electionRoll });
-}
-
 const addElectionRoll = async (req: any, res: any, next: any) => {
```

Comment:
Should this snippet be deleted since it was replace by addElectionRollController.ts?

Reply:
That is still used in the create election route which scott is working on.